models/public/mobilenet-v2-1.4-224/mobilenet-v2-1.4-224.md

```diff
@@ -0,0 +1,62 @@
+# mobilenet-v2-1.4-224
+
+## Use Case and High-Level Description
+
+`mobilenet-v2-1.4-224` is one of MobileNets - small, low-latency, low-power models parameterized to meet the resource constraints of a variety of use cases. They can be built upon for classification, detection, embeddings and segmentation similar to how other popular large scale models are used. For details see [paper](https://arxiv.org/abs/1704.04861)
+
+## Example
+
+## Specification
+
+| Metric                          | Value                                     |
+|---------------------------------|-------------------------------------------|
+| Type                            | Classification                            |
+| GFlops                          | 1.183                                     |
+| MParams                         | 6.087                                     |
+| Source framework                | Tensorflow\*                              |
+
+## Performance
+
+## Input
+
+### Original model
+
+Image, name - `input` , shape - [1x224x224x3], format [BxHxWxC],
+   where:
+
+    - B - batch size
+    - H - image height
+    - W - image width
+    - C - number of channels
+
+   Expected color order - RGB.
+   Mean values - [127.5, 127.5, 127.5], scale factor for each channel - 127.5
```

Comment:
in the config
- --scale_values=input[127.00001206500114]
Where is the right value: in the doc or in the config?